### DISCUSSION TITLE
Addon for Scratch and Turbowarp that automatically deletes "my variable"

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#6635] [Ideas] [answered]
It's annoying having to delete it manually.

**Comment by Samq64** [comment]
This can be done with the customizable default project addon. It isn't on TurboWarp though.

**Reply by DNin01** [reply]
Yeah. It might also be difficult to code this in a way that it doesn't conflict with that addon.

**Reply by BroJac5246** [reply]
> Yeah. It might also be difficult to code this in a way that it doesn't conflict with that addon.

I don't think it would conflict (you just check if the variable exists before you try to delete it). However, it would definitely be redundant.

Theoretically, a TurboWarp-exclusive addon _could_ be made.

**Comment by ghost** [comment]
I don't really use Scratch Addons, but I use Turbowarp's addons...

**Reply by BroJac5246** [reply]
Scratch Addons already has the solution for this, so it makes no sense for us to create a new addon just for TurboWarp, which uses its own default project.

@GarboMuffin (TurboWarp's developer) do you have any thoughts on this? Maybe `my variable` could be removed from TurboWarp's default project? After all, no one probably uses it and TurboWarp users probably know what variables are.

**Reply by WorldLanguages** [reply, marked as answer]
> Scratch Addons already has the solution for this, so it makes no sense for us to create a new addon just for TurboWarp, which uses its own default project.

Actually, we could try to bundle this with #4707

**Reply by BroJac5246** [reply]
> > Scratch Addons already has the solution for this, so it makes no sense for us to create a new addon just for TurboWarp, which uses its own default project.
> 
> Actually, we could try to bundle this with #4707 

I'm not against that. But it still feels somewhat redundant.
